### ISSUE TITLE
compatible with Fastify 3

### ISSUE THREAD

**Issue by mubasshir** [jeffminsungkim/nestjs-multer-extended#193]
my complete project has been migrated to Fastify, the only part which isn't working is this plugin. If we collectively can migrate it to fastify, a lot of users can be benefitted with performance.

**Comment by jeffminsungkim** [comment]
Definitely, I'd like to support Fastify. I need an extra pair of hands to achieve that though. 😟

**Comment by sandeshgb** [comment]
@jeffminsungkim 
any update on this as I'm facing same issue here...

**Comment by mubasshir** [comment]
@jeffminsungkim any update on it? Is there anyway I can be of any help?
